app.js

```diff
@@ -0,0 +1,165 @@
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+
+var pike = {
+  minCust: 23,
+  maxCust: 65,
+  avgCookies: 6.3,
+  hourlySales: []
+};
+
+pike.calcHourlyCookies = function(){
+  for(i = 0; i < hours.length; i++){
+    var pikeCust = Math.floor(Math.random() * (pike.maxCust - pike.minCust + 1)) + pike.minCust;
+    var hourlyCookies = Math.floor(pikeCust * pike.avgCookies);
+    pike.hourlySales.push(hourlyCookies);
+  }
+};
+
+pike.displayList = function(){
+  var total = 0;
+  var pikeList = document.getElementById('FirstAndPike');
+  for(i = 0; i < hours.length; i++){
+    var hourly = document.createElement('li');
+    hourly.textContent = hours[i] + ': ' + pike.hourlySales[i];
+    pikeList.appendChild(hourly);
+    console.log(total += pike.hourlySales[i]);
+  }
+  var totalCookies = document.createElement('li');
+  totalCookies.textContent = 'Total: ' + total;
+  pikeList.appendChild(totalCookies);
+};
+
+pike.calcHourlyCookies();
```

Comment:
You can call methods like this inside of the `pike.displayList` method as part of generating the data before displaying it. Stuff like this shouldn't be in the global scope.